### DISCUSSION TITLE
Gauss Curl Hybrid Model

### DISCUSSION THREAD

**Discussion by Liam1398** [NatLabRockies/floris#199] [Q&A] [answered]
Hello there I'm currently involved in a project using wake steering in windfarms and I wanted ask a question regarding the Gauss Curl Hybrid model. I've seen file called input_gch_paper in the other_jsons folder, in the examples section when I download FLORIS, and was wondering If I place this input file into my FLORIS model am I using the GCH model? 
Any help would be greatly appreciated!

**Comment by bayc** [comment, marked as answer]
Hello, that input file is to re-create specific cases from the GCH journal article. Perhaps the easiest way to ensure you are using the GCH model for your simulations is to build your `input.json` file from the `floris/examples/example_input.json` file. The wake model parameters defined in there are for the GCH model, in particular:

```
"velocity_model": "gauss_legacy",
"wake_velocity_parameters": {
  "gauss_legacy": {
    "calculate_VW_velocities": true,
    "eps_gain": 0.2,
    "ka": 0.38,
    "kb": 0.004,
    "use_yaw_added_recovery": true
  }
},
"deflection_model": "gauss",
"wake_deflection_parameters": {
  "gauss": {
    "dm": 1.0,
    "eps_gain": 0.2,
    "use_secondary_steering": true
  }
}
```

The GCH modifications are most tested/validated with the `gauss_legacy` velocity model and the `gauss` deflection model currently. You can also enable/disable the GCH modifications once you already have a loaded `floris_interface` object by using `floris_interface.set_gch(enable=True)` and changing the `enable` input as needed.

**Reply by Liam1398** [reply]
Thank you very much for the really quick reply! I'll have a lash using that model thanks very much for the help!

**Reply by bayc** [reply]
Happy to help!
